ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -653,6 +1161,22 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P4S Curtain Call Debuffs',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AF4', capture: true }),
+      condition: (data) => Conditions.targetIsYou() && data.act === 'curtain',
+      delaySeconds: (data, matches) => {
+        const duration = parseFloat(matches.duration);
+        return (data.role === 'dps' ? duration - 12 : duration - 6);
```

Comment:
Hmm, I think this whole trigger is a little strat specific.  I could see a warning at ~5 seconds left to break your tether, because at that point you just have to go or it will explode, but 12 and 6 role-specific seems a little too specific to particular strategies about who goes first.

Reply:
How about a customizable timing with default of 6?